uml/state_diagrams/sensor.plantuml

```diff
@@ -0,0 +1,34 @@
+@startuml
+
+skinparam shadowing false
+
+title Sensor State Diagram
+
+state "Sensor Enabled" as E{
+
+    state Choice1 <<choice>> 
+    state "Searching" as S : do: Search for bluetooth\ndevices (background)
```

Comment:
Make sense to repeat the bluetooth state machine here? Could check instead for the bluetooth being enabled?

Reply:
I didn't realise I included this file - it is incomplete. That line is copy-pasted